adi/rx_tx.py

```diff
@@ -106,6 +109,10 @@ def __del__(self):
         self._rxadc = []
 
     def _rx_init_channels(self):
+        for m in self._rx_channel_names:
```

Comment:
This loop will break other devices and ignores the rx_enabled_channels property

Reply:
Yes, I think it will break buffers that were already acquired but not pulled. I didn't think of that. 

However, the problem here is that if you enable a channel there is no way to disable it. Do you think we should loop over all channels at line 114/120 and do enabled true/false if the current iterator is in rx_enabled_channels ?

Reply:
We could probably add a check to see if a buffer exists then destroy it before modifying enabled channels